connector/src/main/java/io/pixelsdb/pixels/trino/PixelsWorkerCoordinator.java

```diff
@@ -0,0 +1,20 @@
+package io.pixelsdb.pixels.trino;
+
+import io.pixelsdb.pixels.common.utils.ConfigFactory;
+import io.pixelsdb.pixels.planner.coordinate.WorkerCoordinateServer;
+
+public class PixelsWorkerCoordinator {
+    private static WorkerCoordinateServer instance;
+
+    private PixelsWorkerCoordinator() {}
+
+    public static void init()
+    {
+        if (instance == null) {
+            int port = Integer.parseInt(ConfigFactory.Instance().getProperty("worker.coordinate.server.port"));
+            instance = new WorkerCoordinateServer(port);
+            Thread workerThread = new Thread(instance);
+            workerThread.start();
```

Comment:
We should terminate the worker coordinator when the Trino process is shutting down.
I think we can try:
1. terminate the worker coordinator server in the shutdown method of PixelsConnector.
2. add a JVM shutdown hook here to terminate the worker coordinator server when the Trino process is shutting down.